### PR TITLE
Update Documentations

### PR DESCRIPTION
**Pull request by pcgiang**
- [x]  update user guide
- [x]  update about us

### CODE REVIEW

**Comment by pcgiang** [comment]
- [ ] update user guide
- [ ] update about us

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,23 +14,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `SoConnec.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```
Comment (janjanchen) [inline docs/UserGuide.md:17]: JAR file name should be SoConnect.jar

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,23 +14,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your SoConnect app.
```
Comment (janjanchen) [inline docs/UserGuide.md:19]: The numbering for quick start should be 1, 2, 3, ...

File: docs/UserGuide.md [hunk 3]
```diff
@@ -14,23 +14,23 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
```
Comment (janjanchen) [inline docs/UserGuide.md:22]: Should we not include the old image from AB3?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -181,12 +183,30 @@ _Details coming soon ..._
 
 ## Command summary
 
+### Manage Contacts
```
Comment (janjanchen) [inline docs/UserGuide.md:186]: Should we name the header as Managing Contacts?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -181,12 +183,30 @@ _Details coming soon ..._
...
+Action | Format, Examples
+--------|------------------
+**Add** | `cadd n/NAME e/EMAIL [p/PHONE_NUMBER] [a/ADDRESS] [th/TELEGRAM_HANDLE] [z/ZOOM] [t/TAG]…​` <br> e.g., `cadd n/James Ho p/22224444 e/hohohojames@u.nus.edu a/123, Clementi Rd, 1234665 t/Professor`
+**Clear** | `cclear`
+**Delete** | `cdelete INDEX1[-INDEX2]`<br> e.g., `cdelete 3` <br> e.g., `cdelete 1-5`
+**Edit** | `cedit INDEX [n/NAME] [e/EMAIL] [p/PHONE] [a/ADDRESS] [th/TELEGRAM_HANDLE] [z/ZOOM] [dt/TAG_DELETED]…​ [t/TAG_ADDED]…​​`<br> e.g.,`cedit 2 n/James Lee e/jameslee@u.nus.edu p/91234567 dt/OP1_projectmate t/CS2103T_projectmate t/roommate` <br> e.g., `cedit 3 dt/*`
+**Find** | `cfind KEYWORD [MORE_KEYWORDS]`<br> e.g., `cfind James Jake`
+**List** | `clist [e] [p] [a] [th] [z] [t]` <br> e.g., `clist` <br> e.g., `clist e p`
+
+### Manage Events
```
Comment (janjanchen) [inline docs/UserGuide.md:197]: Should we name the header as Managing Events?

**Review by janjanchen — CHANGES_REQUESTED** [review]

**Review by janjanchen — APPROVED** [review]

**Pull request merged**